src/func/Microsoft.Health.Fhir.Ingest.Host/IomtConnectorFunctions.cs

```diff
@@ -58,6 +58,10 @@ public static class IomtConnectorFunctions
                 EnsureArg.IsNotNull(events, nameof(events));
 
                 var template = CollectionContentTemplateFactory.Default.Create(templateDefinitions);
+                if (!template.IsValid(out string error))

```

Comment:
This could also be done as an extension method.

Reply:
Updated.